docs/_posts/2017-07-14-ahernandez-intro.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "Community Manager Introduction"
+date:   2017-07-14 16:49:53
+author: Alexander Hernandez
+thumbnail: ahernandez.jpg
+---
+
+Hello Everyone!
+
+My name is Alexander Hernandez and I would like to introduce myself as your OpenGEE Community Manager. I’m currently employed at Thermopylae Sciences and Technology and I’m excited to be a part of this new community. At TST we’re looking forward to improving the project with other Google partners as well as working with the open source community.
+
+As for myself, I worked in the Defense Industry for six years and then transitioned to my current role for the past three. Outside of work you might find me playing guitar in Dallas or out on the golf course on a sunny day. Throughout my career I have been worked with various GIS products so I'm excited to be a part of the community driving OpenGEE.
```

Comment:
"Throughout my career I have been worked with various GIS products so I'm excited to be a part of the community driving OpenGEE." -> "Throughout my career I have worked with various GIS products, so I'm excited to be a part of the community driving Open GEE."